data/scenarios/Challenges/hanoi.yaml

```diff
@@ -63,18 +102,33 @@ entities:
     description:
     - A disk of radius 1.
     properties: [portable]
-  - name: blocked three
+  - name: blocked one
```

Comment:
The blocked one should not be necessary, but with race conditions, it could come up and it's easier to handle it this way.